Lib/extractor/formats/opentype.py

```diff
@@ -146,7 +146,7 @@ def extractGlyphPrograms(source, destination):
         return
     glyph_set = source.getGlyphSet()
     for name in glyph_set.keys():
-        glyph = glyph_set[name]._glyph
+        glyph, offset = glyph_set[name]._getGlyphAndOffset()
```

Comment:
glyph = glyph_set.glyfTable[name]

Reply:
No, just get the `glyf` table from the ttfont. Why even use getGlyphSet at all?

Reply:
`glyph_set.glyfTable` should also be private.